docs/Usage/Response.md

```diff
@@ -29,6 +29,28 @@ def get_book(path: BookPath, query: BookBody):
     return {"code": 0, "message": "ok", "data": {}}
 ```
 
+*New in v2.5.0*
+
+Now you can use `string`, `int`, and `HTTPStatus` as response's key.
+
+```python hl_lines="5 7"
+from http import HTTPStatus
+
+@api.get("/hello/<string:name>",
+        responses={
+            HTTPStatus.OK: Message, 
```

Comment:
Odd typehint. You're annotating HTTPStatus.OK with "Message", but I do not see anywhere this Message class.

I'd assume "Message" would be a Union[str, int, HTTPStatus], is the Message class declared somewhere as a type alias?

e.g.
`Message = Union[str, int, HTTPStatus]`

Reply:
I just found it here:
examples/response_demo.py

But this means the class is inaccurate in the demo. Therefore this file needs changing too.

Reply:
`HTTPStatus.OK: Message` is not a type of hint. `HTTPStatus.OK` is key and `Message` is value.

And I will improve the documentation here.

Reply:
Apologies I was tired and didn't read properly. Please ignore!